### PR TITLE
255 optionset helper

### PR DESCRIPTION
**Pull request by godfreyyeung**
This PR introduces a helper for a standardized way of working with optionsets and their codes and values in templates.

Addresses #255

### CODE REVIEW

File: client/app/helpers/helpers.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+## Helpers
```
Comment (trbmcginnis) [inline client/app/helpers/helpers.md:1]: Love the inclusion of this readme file but as a first-time viewer I'm having trouble understanding it intuitively. Would you mind adding more explanation?

Comment (godfreyyeung) [reply client/app/helpers/helpers.md:1]: Yes, Can do!

File: client/app/helpers/optionset.js [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+  package: {
+    state: PACKAGE_STATE_OPTIONSET,
+    status: PACKAGE_STATUS_OPTIONSET,
+    visibility: PACKAGE_VISIBILITY_OPTIONSET,
+  },
+};
+
+
+/**
+ * Use this helper in templates to retrieve optionsets and their values.
```
Comment (trbmcginnis) [inline client/app/helpers/optionset.js:33]: great comments :)

File: client/app/helpers/helpers.md (hunk 1 above)
Comment (godfreyyeung) [reply client/app/helpers/helpers.md:1]: I added some intro text. Not sure if that helps, let me know what you think can make it clearer!

File: client/app/components/project-geography.js [hunk 3]
```diff
@@ -2,17 +2,17 @@ import Component from '@glimmer/component';
 import { action } from '@ember/object';
 import { inject as service } from '@ember/service';
 import { bblBreakup } from '../helpers/bbl-breakup';
-import { PROJECT_BBL_BOROUGHS } from '../models/bbl';
+import { BOROUGHS_OPTIONSET } from '../models/bbl';
```
Comment (bfreeds) [inline client/app/components/project-geography.js:5]: Nice improvement on the language here.

File: client/app/components/packages/applicant-fieldset.hbs [hunk 4]
```diff
@@ -78,14 +78,14 @@
...
           @placeholder="state"
           @searchEnabled={{true}}
           @searchField="label"
           @options={{this.stateOptions}}
-          @onchange={{fn this.updateAttrWithOption @applicant 'dcpState'}}
-        as |state|>
-          {{state.label}}
+          @onchange={{fn this.updateAttr @applicant 'dcpState'}}
+        as |stateCode|>
+          {{optionset 'applicant' 'state' 'label' stateCode}}
```
Comment (bfreeds) [inline client/app/components/packages/applicant-fieldset.hbs:88]: Amazing ^

File: client/app/helpers/helpers.md (hunk 1 above)
Comment (bfreeds) [reply client/app/helpers/helpers.md:1]: Thank you for your thoughtfulness of developer experience in including this

File: client/app/components/packages/applicant-fieldset.hbs (hunk 4 above)
Comment (godfreyyeung) [reply client/app/components/packages/applicant-fieldset.hbs:88]: 👍 :)

**Pull request merged**